docs/en/setup/service-agent/java-agent/agent-optional-plugins/Logger-plugin.md

```diff
@@ -0,0 +1,49 @@
+# Overview
+
+`logger-plugin` can store the logs generated by the program during the call, such as the content of the error log, into the span log.  Through the configuration file, you can control the log source (log4j2, logback, log4j), package name, and level.
+
+# Configuration file
+
+By default, the configuration file is in `apache-skywalking-apm-bin/agent/config//logger-plugin/logconfig.properties`.Of course, **If the file does not exist, the configuration will still be configured with the following default values:**
```

Comment:
According to this, we should not have the default `logconfig.properties` in the distribution.

Reply:
If the configuration file not exists, what should we do? Throws an Exception?

Reply:
I have removed `logconfig.properties` and add some use cases in the document.